tests/unit/test_offers.py

```diff
@@ -24,10 +24,6 @@ def basic_offer(self):
     def bare_metal_offer(self):
         return AWSOffer(BARE_METAL_EC2_OFFER)
 
-    def test_raw(self, offer):
```

Comment:
We'll want to document this in release notes, as it may affect some users

Reply:
On that note, will this be v2.0.0 since it's not backwards compatible? Will update PR with release notes as well

Reply:
Yep, I'd recommend just bumping straight to `2.0.0`

Reply:
Updated version and change log, although we don't actually remove `.raw` so I added the test back in